pilot/pkg/serviceregistry/kube/controller/filter/discoverynamespaces.go

```diff
@@ -74,6 +74,10 @@ func (d *discoveryNamespacesFilter) Filter(obj any) bool {
 		return true
 	}
 
+	if ns, ok := obj.(string); ok {
```

Comment:
id like to avoid this if possible. Will see what that looks like when I get to the other parts of the PR...

Reply:
seems feasible?

Reply:
removed